src/Microsoft.Health.Fhir.SqlServer/Features/Schema/SchemaVersionConstants.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.Health.Fhir.SqlServer.Features.Schema
     public static class SchemaVersionConstants
     {
         public const int Min = (int)SchemaVersion.V4;
-        public const int Max = (int)SchemaVersion.V28;
+        public const int Max = (int)SchemaVersion.V29;
```

Comment:
I remember we need to change some tests for version, do we run all tests on local?

Reply:
Tested algready